jmix-quartz/quartz-ui/src/main/resources/io/jmix/quartzui/messages.properties

```diff
@@ -52,3 +52,14 @@ io.jmix.quartzui.screen.trigger/cronExpressionHelpText=Cron expression is a sequ
                     <li>0 11 11 11 11 ? - fires every November 11th at 11:11 AM</li>\
                     <li>0 0 0 25 12 ? - fires every Christmas at midnight</li>\
                 </ul>
+io.jmix.quartzui.screen.trigger/RepeatMode.NO_REPEATS=No repeats
```

Comment:
Maybe we can name this "Single execution" or "Run once" or "One-time trigger" or something like this? Won't these names better explain what will happen?

Reply:
Changed